nss-day-cohort-51/nss-day-cohort-51.github.io
# mw-TechStack-list-update-111 Updated the TechStack array object in th…

…e database; doublechecked devType properties have the correct developer type or All values.

# Description
Updated the final copy of the Tech Stack list in the database.

## Type of change


# Testing Instructions
Serve the website with npm start .
Go to Tech Stack section on the webpage and see the following tech totals:

- **All** should have **19** tech stack items 
- **UI/UX FRONT-END** should have **11** tech stack items 
- **FULL STACK** should have **8** tech stack items 

ADDITION:
Added a "More Details" button on the Event Card.
Added styling to fall in line with existing buttons.

# Checklist:
